src/scrollTo.js

```diff
@@ -95,6 +100,24 @@ export const scroller = () => {
     if (abort) return done()
     if (!timeStart) timeStart = timestamp
 
+    // When a site has a lot of media that can be loaded asynchronously,
+    // the targetY/targetX may end up in the wrong place during scrolling.
+    // So we will check this at each step
+    if (dynamicPosition) {
+      cumulativeOffsetContainer = _.cumulativeOffset(container)
+      cumulativeOffsetElement = _.cumulativeOffset(element)
+      if (x) {
+        targetX =
+          cumulativeOffsetElement.left - cumulativeOffsetContainer.left + offset
+        diffX = targetX - initialX
+      }
+      if (y) {
+        targetY =
+          cumulativeOffsetElement.top - cumulativeOffsetContainer.top + offset
+        diffY = targetY - initialY
+      }
```

Comment:
I would refactor this since calculations are same as https://github.com/rigor789/vue-scrollto/blob/eb11b5feaad76965b13dc2ba5c8b6471b2210ef0/src/scrollTo.js#L196-L207

Refactoring both places to use a function would likely be a cleaner approach - since if we ever need to change how the target is calculated, it needs to be done in a single spot. 

Something like `recalculateTargets`, or something better perhaps.

Reply:
I probably won't be able to for at least a few weeks, so if you are able to do it, that would be perfect!

Reply:
Done